### PR TITLE
Add decode to build_paper output to make it format better

### PR DESCRIPTION
**Pull request by ksunden**
Was printing Byte-string, which was horrible to read

### CODE REVIEW

File: publisher/build_paper.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def tex2pdf_singlepass(out_path):
     if b"Fatal" in out or run.returncode:
         print("PDFLaTeX error output:")
         print("=" * 80)
-        print(out)
+        print(out.decode())
```
Comment (mpacer) [inline publisher/build_paper.py]: My intuition is that we should decode it to `'utf-8'` for consistency with Python3 (which is currently planned to be the only supported python as of next year). Is there a reason not to do that?

**Comment by mpacer** [comment]
🎉 For making this PR against `dev`. You rock!

**Comment by deniederhut** [comment]
I would like to second to motion for you rocking.

**Comment by deniederhut** [comment]
1pfr

**Pull request merged**